server/src/test/scala/services/crunch/TransferPaxInApiSpec.scala

```diff
@@ -75,7 +75,7 @@ class TransferPaxInApiSpec extends CrunchTestLike {
     success
   }
 
-  "Given a flight with transfer passengers " +
+  "Given a flight with transfer passengers and no live feed pax numbers " +
     "Then these passengers should not be included in the total pax when using API pax nos" >> {
```

Comment:
This test description was wrong before you edited it. It appears to be testing that transfer pax are taken off the API pax figure in the case where there are no flight feed pax nos, but there transfer number is zero so it's not actually testing that

Reply:
For this incorrect test description we could either amend it so it doesn't say it's checking that trans pax are removed from the api pax nos, or add some trans pax so it does what the description says

Reply:
so it does actually have transfer passengers - they are just specified in the manifests, not in the live feed - I'll improve the description to make that obvious.